Examples/Examples/FeatureFormExampleView.swift

```diff
@@ -61,18 +64,38 @@ struct FeatureFormExampleView: View {
                     ArcGISEnvironment.authenticationManager.arcGISCredentialStore.add(publicSample!)
                 }
                 .ignoresSafeArea(.keyboard)
-                .floatingPanel(
-                    attributionBarHeight: attributionBarHeight,
-                    selectedDetent: $detent,
-                    horizontalAlignment: .leading,
-                    isPresented: model.formIsPresented
-                ) {
-                    if let featureForm = model.featureForm {
+            // NavigationStack doesn't work properly inside of a FloatingPanel
+            // so we need to use a sheet instead for now.
+            // Similar to https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/issues/706
```

Comment:
Just [asked](https://developer.apple.com/forums/thread/772650) about this on the Apple Developer Forums.